src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Experimental Features'
+metaDescription: "Opt-in to use experimental features in New Relic Browser before they are generally available."
+redirects:
+  - /docs/browser/new-relic-browser/configuration
+freshnessValidatedDate: never
+---
+
+New Relic Browser features are exposed to customers in a controlled manner to ensure they are stable and reliable. However, some features are exposed before they generally available. These features are called **experimental features**. To access these features, you must opt in to use them.
+
+## Current experimental features
+
+The following experimental features are available in New Relic Browser:
+* Browser Agent v1.272.0 - [Automatically track native marks and measures as `BrowserPerformance` events](/docs/browser/new-relic-browser/browser-pro-features/marks-and-measures).
+* Browser Agent v1.276.0 - [Automatically observe page resource assets as `BrowserPerformance` events](/docs/browser/new-relic-browser/browser-pro-features/page-resources).
+
+
+<Callout variant="important">
+  Experimental features are only available for opt-in on copy/paste or NPM implementations of the agent. These features are subject to change and should be used with caution.
```

Comment:
Experimental features are available only for opt-in on manual copy and paste, or NPM implementations of the agent. These features are subject to change and should be used with caution.